pkg/queue/queue.go

```diff
@@ -179,6 +179,11 @@ func (q *Queues) GetQueueInfo(namespace string, name string) (string, int32, int
 }
 
 func parseQueueURI(uri string) (string, string, error) {
+
+	if strings.Contains(uri, "bs") {
```

Comment:
This is not required if queueURI is a valid URL with scheme `beanstalk`. Eg: `beanstalk://127.0.0.1:11300/test-tube`

This will match false-positives for cases like `https://sqs.ap-south-1.amazonaws.com/1234567890/test-bs-tube`

Reply:
I was testing the PR and hence I made the changes in a new branch and gave a PR [here](https://github.com/Exotel-Org/k8s-worker-pod-autoscaler/pull/1).

Reply:
Thanks. Merged it